### DISCUSSION TITLE
How to get a lager page width in Digital Garden instead of center

### DISCUSSION THREAD

**Discussion by trinityzhub** [oleeskild/obsidian-digital-garden#653] [General]
How to get a lager page width in Digital Garden  like “Readable line length” in Obsidian

**Comment by trinityzhub** [comment]
we can do similar thing in obsidian app by disabling  “Readable line length”  in settings of obsidian


https://forum.obsidian.md/t/how-to-get-a-lager-page-width-in-both-editing-mode-and-preview-mode/7555

**Comment by olaffthunder** [comment]
I'd like to know that, too.
I got a dataview table and it's horrible to read in the current content width.  :/
